### ISSUE TITLE
Cargo vomits when depending on `glium` and `image at the same time.

### ISSUE THREAD

**Issue by Zarathustra30** [rust-lang/cargo#2139]
Whenever I run "cargo build", I get a ridiculously long error message, ending with ```thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: Any', ../src/libcore\result.rs:732".```Unfortunately, the error message gets cut off.

Currently compiling with Rust 1.4 / Cargo 0.5 on Windows 10 by way of multirust-rs. I am using gcc 5.2.0 supplied by Cygwin.

Cargo.toml:
```
[package]
name = "my_project"
version = "0.1.0"

[dependencies]
glium = "*"
image = "*"
```
Cargo.lock:
```
[root]
name = "my_project"
version = "0.1.0"
dependencies = [
 "glium 0.11.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "image 0.5.0 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "advapi32-sys"
version = "0.1.2"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "android_glue"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "backtrace"
version = "0.1.8"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "backtrace-sys 0.1.3 (registry+https://github.com/rust-lang/crates.io-index)",
 "cfg-if 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "dbghelp-sys 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "debug-builders 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "kernel32-sys 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "backtrace-sys"
version = "0.1.3"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "bitflags"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "bitflags"
version = "0.3.2"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "byteorder"
version = "0.3.13"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "cfg-if"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "cgl"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "gleam 0.1.16 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "cgmath"
version = "0.4.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
 "rand 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "rustc-serialize 0.3.16 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "cocoa"
version = "0.1.5"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bitflags 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "core-graphics 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "objc 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "color_quant"
version = "1.0.0"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "core-foundation"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "core-foundation-sys 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "core-foundation-sys"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "core-graphics"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "core-foundation 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "dbghelp-sys"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "debug-builders"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "dlib"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "dwmapi-sys"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "dylib"
version = "0.0.2"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "enum_primitive"
version = "0.0.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "enum_primitive"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "flate2"
version = "0.2.11"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "miniz-sys 0.1.7 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gcc"
version = "0.3.19"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "advapi32-sys 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gdi32-sys"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gif"
version = "0.5.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "color_quant 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "lzw 0.8.0 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gif"
version = "0.6.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "color_quant 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "lzw 0.9.0 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gl_common"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gl_generator"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "khronos_api 0.0.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "log 0.3.3 (registry+https://github.com/rust-lang/crates.io-index)",
 "xml-rs 0.1.26 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gl_generator"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "khronos_api 0.0.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "log 0.3.3 (registry+https://github.com/rust-lang/crates.io-index)",
 "xml-rs 0.1.26 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "gleam"
version = "0.1.16"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "gl_common 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "gl_generator 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "khronos_api 0.0.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "glium"
version = "0.11.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "backtrace 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "cgmath 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "gl_generator 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "glutin 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "image 0.4.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "khronos_api 0.0.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "lazy_static 0.1.15 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "nalgebra 0.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "smallvec 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "glob"
version = "0.2.10"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "glutin"
version = "0.4.2"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "android_glue 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "cgl 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "cocoa 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "core-foundation 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "core-graphics 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "dwmapi-sys 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "gdi32-sys 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "gl_common 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "gl_generator 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "kernel32-sys 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "khronos_api 0.0.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "lazy_static 0.1.15 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "objc 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
 "osmesa-sys 0.0.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "shared_library 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "shell32-sys 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "user32-sys 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "wayland-client 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "wayland-kbd 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "wayland-window 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "x11-dl 2.0.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "image"
version = "0.4.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "byteorder 0.3.13 (registry+https://github.com/rust-lang/crates.io-index)",
 "enum_primitive 0.0.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "gif 0.5.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "glob 0.2.10 (registry+https://github.com/rust-lang/crates.io-index)",
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
 "png 0.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "image"
version = "0.5.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "byteorder 0.3.13 (registry+https://github.com/rust-lang/crates.io-index)",
 "enum_primitive 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "gif 0.6.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "glob 0.2.10 (registry+https://github.com/rust-lang/crates.io-index)",
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
 "png 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "inflate"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "kernel32-sys"
version = "0.2.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "khronos_api"
version = "0.0.8"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "lazy_static"
version = "0.1.15"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "libc"
version = "0.1.12"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "libc"
version = "0.2.2"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "log"
version = "0.3.3"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "lzw"
version = "0.8.0"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "lzw"
version = "0.9.0"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "malloc_buf"
version = "0.0.6"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "miniz-sys"
version = "0.1.7"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "gcc 0.3.19 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "mmap"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "tempdir 0.3.4 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "nalgebra"
version = "0.3.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
 "rand 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "rustc-serialize 0.3.16 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "num"
version = "0.1.27"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "rand 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "rustc-serialize 0.3.16 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "objc"
version = "0.1.8"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "malloc_buf 0.0.6 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "osmesa-sys"
version = "0.0.5"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "shared_library 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "png"
version = "0.3.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bitflags 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "flate2 0.2.11 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "png"
version = "0.4.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bitflags 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "flate2 0.2.11 (registry+https://github.com/rust-lang/crates.io-index)",
 "inflate 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "num 0.1.27 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "rand"
version = "0.3.12"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "advapi32-sys 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "rustc-serialize"
version = "0.3.16"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "shared_library"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "lazy_static 0.1.15 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "shell32-sys"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "smallvec"
version = "0.1.5"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "tempdir"
version = "0.3.4"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "rand 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "tempfile"
version = "1.1.2"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "kernel32-sys 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "rand 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "user32-sys"
version = "0.1.2"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi-build 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "wayland-client"
version = "0.2.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bitflags 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "dlib 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "lazy_static 0.1.15 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "wayland-kbd"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bitflags 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "lazy_static 0.1.15 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.1.12 (registry+https://github.com/rust-lang/crates.io-index)",
 "mmap 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "wayland-client 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "wayland-window"
version = "0.1.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "byteorder 0.3.13 (registry+https://github.com/rust-lang/crates.io-index)",
 "tempfile 1.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "wayland-client 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "winapi"
version = "0.2.5"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "winapi-build"
version = "0.1.1"
source = "registry+https://github.com/rust-lang/crates.io-index"

[[package]]
name = "x11-dl"
version = "2.0.1"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "dylib 0.0.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
]

[[package]]
name = "xml-rs"
version = "0.1.26"
source = "registry+https://github.com/rust-lang/crates.io-index"
dependencies = [
 "bitflags 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
]
```

**Comment by svip** [comment]
I can also reproduce this error.  It appears to happen when compiling with `glium` 0.11 and `image` 0.5.  You can fix it by relying on `image` 0.4 instead.

The incident is more related to this line: https://github.com/rust-lang/cargo/blob/master/src/cargo/core/registry.rs#L103 than the unwrap.

**Comment by alexcrichton** [comment]
I believe that this is a dupe of https://github.com/rust-lang/cargo/issues/2132, so closing in favor of that (some comments at the end of the thread there)
